__tests__/components/forms/TeamMemberForm.test.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import TeamMemberForm from '../../../front-end/src/components/forms/TeamMemberForm';
+import teamMembersData from '../../fixtures/teamMembers';
+import teamsData from '../../fixtures/teams';
+import rolesData from '../../fixtures/roles';
+
+test('should render TeamMemberForm correctly', () => {
+    const wrapper = shallow(
+        <TeamMemberForm
+            title="Test Team Form"
+            teamMember={teamMembersData[0]}
+            teams={teamsData}
+            roles={rolesData}/>
+    );
+    expect(wrapper).toMatchSnapshot()
+});
+
+test('should render invalid TeamMemberForm correctly', () => {
```

Comment:
The names of these tests should be more specific, rather than having a series of tests with the same name.

Reply:
I need you to be more specific what you mean by that. Give an example